code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -60,6 +60,26 @@
 				message = text("<span class='name'>[src]</span> signs[].", (text2num(param) ? text(" the number []", text2num(param)) : null))
 				m_type = 1
 
+		if ("special")	//visual emotes with their own special sprites - DO NOT BE BLIND LIKE A CERTAIN SOMEONE WHO SHOULD HAVE ASKED WHAT EACH SPRITE WAS

```

Comment:
If you do not know, try, that is how you learn to meme as well as anything else.